PL/Moodles_PL.txt

```diff
@@ -0,0 +1,114 @@
+  Moodles_endurance_desc_lvl2 = "Wyczerpanie 50%^<br>zmniejszone i ciemniejsze pole widzenia w trybie zbieractwa<br>-80% obrażeń wręcz<br>-14% szybkości ataku<br>+20% szansy na potknięcie podczas biegu i skakania<br>-10% szansy na wspinanie się po wysokich płotach<br>-37% prędkości poruszania<br>-4% szansy na blokowanie ataków zombi od przodu<br>nie można biec",
+  Moodles_endurance_desc_lvl3 = "Wyczerpanie 75%^<br>okropne i ciemniejsze pole widzenia w trybie zbieractwa<br>-90% obrażeń wręcz<br>-21% szybkości ataku<br>+30% szansy na potknięcie podczas biegu i skakania<br>-15% szansy na wspinanie się po wysokich płotach<br>-56% prędkości poruszania<br>-6% szansy na blokowanie ataków zombiaków od przodu<br>nie można biec<br>nie można sprintować<br>nie można szybko chodzić<br>nie można ćwiczyć",
+  Moodles_endurance_desc_lvl4 = "Wyczerpanie 90%^<br>okropne i ciemniejsze pole widzenia w trybie zbieractwa<br>-95% obrażeń wręcz<br>-28% szybkości ataku<br>+40% szansy na potknięcie podczas biegu i skakania<br>-20% szansy na wspinanie się po wysokich płotach<br>-75% prędkości poruszania<br>-8% szansy na blokowanie ataków zombiaków od przodu<br>nie można biec<br>nie można sprintować<br>nie można szybko chodzić<br>nie można ćwiczyć<br>nie można używać ciężkich broni",
+  -- Heavy load
+  Moodles_heavyload_desc_lvl1 = "100%^ Powyżej pojemności postaci<br>-5% szans na krytyczne trafienie<br>-7% szybkość ataku<br>+13% szansa na potknięcie się podczas przeskakiwania przeszkód<br>-8% szansy na potknięcie podczas biegu i skakania<br>-19% prędkość poruszania<br>-2% szansa na zablokowanie ataków zombie od przodu<br>zmniejszona regeneracja wytrzymałości",
+  Moodles_heavyload_desc_lvl2 = "125%^ Powyżej pojemności postaci<br>-10% szans na krytyczne trafienie<br>-14% szybkość ataku<br>+26% szansa na potknięcie się podczas przeskakiwania przeszkód<br>-16% szansy na potknięcie podczas biegu i skakania<br>-37% prędkość poruszania<br>-4% szansa na zablokowanie ataków zombie od przodu<br>znacznie zmniejszona regeneracja wytrzymałości<br>nie można sprintować",
+  Moodles_heavyload_desc_lvl3 = "150%^ Powyżej pojemności postaci<br>-15% szans na krytyczne trafienie<br>-21% szybkość ataku<br>+39% szansa na potknięcie się podczas przeskakiwania przeszkód<br>-24% szansy na potknięcie podczas biegu i skakania<br>-56% prędkość poruszania<br>-6% szansa na zablokowanie ataków zombie od przodu<br>brak regeneracji wytrzymałości<br>nie można sprintować<br>nie można ćwiczyć<br>tracenie punktów zdrowia",
+  Moodles_heavyload_desc_lvl4 = "175%^ Powyżej pojemności postaci<br>-20% szans na krytyczne trafienie<br>-28% szybkość ataku<br>+52% szansa na potknięcie się podczas przeskakiwania przeszkód<br>-32% szansy na potknięcie podczas biegu i skakania<br>-75% prędkość poruszania<br>-8% szansa na zablokowanie ataków zombie od przodu<br>tracenie wytrzymałości<br>nie można sprintować<br>nie można ćwiczyć<br>tracenie punktów zdrowia".
```

Comment:
Wrong symbol used here which breaks the formatting. It should be a `,` instead of a `.`.

```suggestion
  Moodles_heavyload_desc_lvl4 = "175%^ Powyżej pojemności postaci<br>-20% szans na krytyczne trafienie<br>-28% szybkość ataku<br>+52% szansa na potknięcie się podczas przeskakiwania przeszkód<br>-32% szansy na potknięcie podczas biegu i skakania<br>-75% prędkość poruszania<br>-8% szansa na zablokowanie ataków zombie od przodu<br>tracenie wytrzymałości<br>nie można sprintować<br>nie można ćwiczyć<br>tracenie punktów zdrowia",
```